src/charm.py

```diff
@@ -106,20 +103,12 @@ def _reconcile(self) -> None:
             kubeconfig = KubeConfig.from_service_account()
             client = Client(config=kubeconfig, field_manager=self.app.name)
         except ConfigError as exc:
-            logger.exception("Error initializing the lightkube client.")
+            logger.error("Error initializing the lightkube client: %s", exc)
```

Comment:
I think `%r` might be better than `%s` to format exceptions, as `%s` will only keep the message of the exception, without the exception class.